pym/_emerge/actions.py

```diff
@@ -1646,6 +1646,10 @@ def action_info(settings, trees, myopts, myfiles):
 		last_sync = portage.grabfile(os.path.join(repo.location, "metadata", "timestamp.chk"))
 		if last_sync:
 			append("Timestamp of repository %s: %s" % (repo.name, last_sync[0]))
+		last_sync = portage.grabfile(os.path.join(repo.location, "metadata", "timestamp.commit"))
+		if last_sync:
+			head_commit = last_sync[0].split()[0]
```

Comment:
This will raise IndexError if the first line is empty for some reason, so we should handle that.